.github/workflows/flake8.yml

```diff
@@ -0,0 +1,24 @@
+name: Flake8 Check
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - name: Install dependencies
+        run: pip install flake8
```

Comment:
There is an action for flake8, wouldn't be better to use it? (https://github.com/py-actions/flake8) or why not?